tests/view_tests/tests/test_i18n.py

```diff
@@ -314,6 +314,16 @@ def test_jsi18n_fallback_language(self):
             self.assertContains(response, 'il faut le traduire')
             self.assertNotContains(response, "Untranslated string")
 
+    def test_i18n_fallback_language_plural(self):
+        """
+        Make sure fallback to a language with less plural forms always maintains
+        the real language's number of plural forms.
+        """
+        with self.settings(LANGUAGE_CODE='pt'), override('ru'):
+            response = self.client.get('/jsi18n/')
+            catalog = response.context['catalog']
+            self.assertEqual(len(catalog['{count} plural3']), 3)
```

Comment:
`self.assertEqual(catalog['{count} plural3'], ['{count} plural3', '{count} plural3s', '{count} plural3 p3t'])` may help debug the test more easily if it fails.